test-requirements.txt

```diff
@@ -1,25 +1,16 @@
-#
-# This file is autogenerated by pip-compile with Python 3.8
-# by the following command:
-#
-#    pip-compile test-requirements.in
-#
+# This file was autogenerated by uv v0.1.1 via the following command:
+#    uv pip compile --python-version=3.8 test-requirements.in -o test-requirements.txt
 alabaster==0.7.13
     # via sphinx
 astor==0.8.1
-    # via -r test-requirements.in
 astroid==3.0.2
     # via pylint
 async-generator==1.10
-    # via -r test-requirements.in
 attrs==23.2.0
-    # via
-    #   -r test-requirements.in
-    #   outcome
+    # via outcome
 babel==2.14.0
     # via sphinx
-black==24.1.1 ; implementation_name == "cpython"
-    # via -r test-requirements.in
+black==24.1.1
```

Comment:
Huh, this change seems bugged

Reply:
Made a bug report https://github.com/astral-sh/uv/issues/1429